akka-docs/src/main/paradox/typed/cqrs-durable-state.md

```diff
@@ -0,0 +1,6 @@
+# CQRS
```

Comment:
Same here on something that separates it for the es cqrs page and crosslinking between the two.

Reply:
done ..